packages/RatPack.package/RPTestApp.class/instance/handleRequestForCaptureGroupsWithNumber.andWord..st

```diff
@@ -0,0 +1,6 @@
+routes
+handleRequestForCaptureGroupsWithNumber: aNumber andWord: aWord
+	<get: '/captureGroups/number/([\d]+)/word/([\w]+)'>
```

Comment:
the route could be just (\d+) and (\w+), you don't need the [] here afaik,

Reply:
Ok